Snakefile

```diff
@@ -268,6 +268,8 @@ rule differential_tracks:
 rule differential_gsea:
     conda: "envs/irap.yaml"
     log: "logs/{accession}.{contrast_id}.{ext_db}-differential_gsea.log"
+    resources: mem_mb=get_mem_mb
+    threads: 8
```

Comment:
Does this get injected into the resources requested to the cluster? Or that would need to be done as well within the resources field above?